test/lib/analyzer/image-inspector.test.ts

```diff
@@ -141,10 +141,6 @@ test("get image as an archive", async (t) => {
       fs.existsSync(path.join(imageSavePath, "image.tar")),
       "image should not exists on disk",
     );
-    t.notOk(
```

Comment:
I need to remove because now the folder is remove asynchrony. So we can't guarantee that it has been already removed when we check it.

Reply:
I think there is a way to test this with Jest/sinon by doing the following:
1. In the code where you call `fs.rmdir(imageFolderPath, (err) => {` you pull out the anonymous  function `(err) => {}` and put it in a separate file and export it -- this makes it possible that we can mock it with Jest/sinon and detect when it is being called.
2. Modify the tests here to mock the exported function and detect when it is being called. Once it's called, we can check the folder really does not exist on disk.

Reply:
Ok! I'm going to do it later! :)